src/App.jsx

```diff
@@ -1,5 +1,30 @@
-function App() {
-  return <div>Hello world!</div>;
-}
+import React, { Suspense } from 'react';
+import { Route, Routes } from 'react-router-dom';
+
+// Components
+import Layout from './hoc/Layout';
+
+// Lazy components
+const Home = React.lazy(() => import('./pages/Home'));
+const About = React.lazy(() => import('./pages/About'));
+
+const App = () => {
+  const routes = [
```

Comment:
would be better to define routes config outside of component body to improve performance